python/convert.py

```diff
@@ -0,0 +1,10 @@
+import mgp
+from json import loads
+
+
+@mgp.function
+def str2object(string: str) -> mgp.Any:
+    if string:
+        return loads(string)
+    else:
+        return None
```

Comment:
here you can just make it like this:
```suggestion
      return None
```

Reply:
Since you are returning on early line, you can return here too same way